pyfr/solvers/baseadvecdiff/elements.py

```diff
@@ -141,7 +141,7 @@ def gradcoru_qpts():
                 'shocksensor', tplargs=tplargs_artvisc, dims=[self.neles],
                 u=self.scal_upts[uin], artvisc=self.artvisc
             )
-        elif shock_capturing == 'none':
+        elif shock_capturing in {'entropy-filter', 'none'}:
```

Comment:
`!= 'artificial-viscosity'`

Reply:
I think that would avoid throwing an error for invalid options?

Reply:
Is this the primary place where we do checking for validity (can't remember if it is here or in the interfaces)?

Reply:
Seems like it